content/use-cases/tokenization/stablecoin-issuer.md

```diff
@@ -4,504 +4,173 @@ parent: tokenization.html
+- Various settings and metadata. Each of the two accounts can control its own settings on the trust line. Each side sets a limit on the trust line.
 
-<!-- SPELLING_IGNORE: fatf, cft -->
+Each trust line is specific to a given currency code. Two accounts can have any number of trust lines between them for different currency codes, but only one shared trust line for any particular currency code.
 
-### Source of Funds
+See [Trust Lines](ft-index.html#trust-lines).
 
-To prevent illicit funds from passing through their systems, financial institutions must be able to determine within reason if the source of a customer’s funds is linked to criminal activity.
+### Authorized Trust Lines
 
-Determining the exact source of funds for every customer may not be administratively feasible. As a result, some regulatory authorities may not provide specific regulation or guidance for all accounts. In specific cases, however, authorities may require financial institutions to identify and report the source of funds. Guidance by the FATF recommends that where the risks of money laundering or terrorist financing are higher (commonly referred to as a “risk-based approach”), financial institutions conduct enhanced due diligence, including but not limited to determining the customer’s source of funds.
+The Authorized Trust Lines feature enables issuers to create tokens that can only be held by accounts that the issuer specifically authorizes. This feature only applies to tokens, not XRP.
 
-<!-- STYLE_OVERRIDE: feasible -->
+To use the Authorized Trust Lines feature, enable the `Require Auth` flag on your issuing account. While the setting is enabled, other accounts can only hold tokens you issue if you have authorized those accounts' trust lines to your issuing account.
 
-### Suspicious Activity Reporting
+See [Authorized Trust Lines](authorized-trust-lines.html).
 
-If a financial institution suspects that funds may be related to criminal activity, the institution must file a Suspicious Activity Report (SAR) with the appropriate regulatory authority. Failure to report suspicious activity may result in in penalties for the institution.
 
-See also:
+### Freeze a Trust Line
 
-- [Suspicious Activity Reporting Overview (USA FFIEC)](https://bsaaml.ffiec.gov/manual/RegulatoryRequirements/04_ep)
+If you issue tokens in the XRP Ledger, you can enable the _No Freeze_ setting to permanently limit your own ability to use the token freezing features of the XRP Ledger. (As a reminder, this only applies to issued tokens, not XRP.) 
 
-- [FATF Recommendation 16: Reporting of suspicious transactions and compliance](http://www.fatf-gafi.org/publications/fatfrecommendations/documents/fatf-recommendations.html)
+If you do not enable the _No Freeze_ setting, when an account shows suspicious activity or violates your institution's terms of use, you have the option of freezing the trust line while you resolve the issue.
```

Comment:
I think it would make more sense to reverse the order of these paragraphs (and rephrase accordingly). For example:

> As a token issuer, when an account shows suspicious activity or violates your terms of use, you have the option of freezing your trust line to that account while you resolve the issue. (As a reminder, this only applies to issued tokens, not XRP.)
>
> To make your tokens more "cash-like" and provide assurances to users that you won't misuse these features, you can enable the _No Freeze_ setting to permanently limit your own ability to use the token freezing features of the XRP Ledger.
```